commercial-partners/index.html

```diff
@@ -63,6 +63,7 @@ <h5>Commercial licenses of OME software</h5>
                     <li><a href="https://www.imagescience.de/imagic.html" target="_blank">Imagic, GmBH</a></li>
                     <li><a href="http://www.perkinelmer.com/" target="_blank">PerkinElmer</a></li>                
                     <li><a href="http://www.mayachitra.com/" target="_blank">Mayachitra Inc.</a></li>
+                    <li><a href=" https://www.yokogawa.com/solutions/products-platforms/life-science/" target="_blank">Yokogawa</a></li>
```

Comment:
I assume this space between " and the URL is why Travis is complaining about this